package.json

```diff
@@ -5,14 +5,56 @@
   "main": "lib/shopify.js",
   "repository": "git@github.com:Shopify/js-buy-sdk.git",
   "scripts": {
-    "build": "node ./scripts/build",
-    "prepublish": "node ./scripts/prepublish",
+    "start": "npm run dir-setup-dist-test && npm run dir-setup-dist && concurrently 'npm run watch-test' 'npm run build-test-sdk' 'http-server -p 4200 .dist-test/'",
```

Comment:
since `start`, `test`, `test-examples`, and `examples` only use `.dist-test` ... may be we don't need `npm run dir-setup-dist`?